### PR TITLE
docs(entropy): write your first entropy app on evm tutorial

### PR DESCRIPTION
**Pull request by 0xfirefist**

### CODE REVIEW

File: pages/entropy/create-your-first-entropy-app.mdx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+```bash copy
+   fee          : 101
+   tx           : 0x9faf2f439f99986b4a87befbad89301b3ec8976038a8b8332c7c53cbba43fd95
+   sequence     : 36
+   tx           : 0x806900dbbebc8488ac6b87c9c82c5fd9ad38231c2e66b1899e0a8a3afe805f01
+   Flip Result  : Tails
+```
+
+Note that: the script can fail due to some api errors. You can run the script again to get the expected result.
```
Comment (m30m) [inline pages/entropy/create-your-first-entropy-app.mdx:289]: rpc issues sounds better. Fortuna should not return errors :D

File: pages/entropy/create-your-first-entropy-app.mdx [hunk 2]
```diff
@@ -0,0 +1,298 @@
+# Create your first Entropy app on EVM
+
+In this tutorial we will implement and deploy a coin flip contract which will use entropy to generate a random output. 
+
+## **Preliminaries**
```
Comment (m30m) [inline pages/entropy/create-your-first-entropy-app.mdx:5]: No need for `**`

File: pages/entropy/create-your-first-entropy-app.mdx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+Create a `script.js` file in `app` and add the following code to the script.
+
+```javascript copy
+const { Web3 } = require("web3");
+const CoinFlipAbi = require("../contracts/out/CoinFlip.sol/CoinFlip.json");
+const EntropyAbi = require("@pythnetwork/entropy-sdk-solidity/abis/IEntropy.json");
+
+async function main() {
+	const web3 = new Web3(process.env["RPC_URL"]);
+  const { address } = web3.eth.accounts.wallet.add(process.env["PRIVATE_KEY"])[0];
```
Comment (m30m) [inline pages/entropy/create-your-first-entropy-app.mdx:186]: indentation issue

File: pages/entropy/create-your-first-entropy-app.mdx [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+Next, add the following code to the main method to request a flip from the CoinFlip contract.
+
+```javascript copy
+async main() {
+	...
+	// Request a random number
+
+	// Generate user random number and commitment
+	const userRandomNumber = web3.utils.randomHex(32);
+  const userCommitment = web3.utils.keccak256(userRandomNumber);
```
Comment (m30m) [inline pages/entropy/create-your-first-entropy-app.mdx:217]: indentation issue

**Review by m30m — APPROVED** [review]

File: pages/entropy/create-your-first-entropy-app.mdx [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+[⠔] Compiling 28 files with 0.8.23
+[⠑] Solc 0.8.23 finished in 3.40s
+Compiler run successful!
+Deployer: 0xfa57d0f2CBDA2729273F2a431E4FeDAc656d0402
+Deployed to: 0x8676ba0Dd492AB9813BC21D5Dce318427d1d73ae
+Transaction hash: 0x2178aa6d402c94166a93e81822248d00dd003827675ebd49b3c542970f5a0189
+```
+
+Let’s export the coin flip contract address as environment variable for later use:
+
```
[the remark this answers is not in the document]
Comment (0xfirefist) [reply pages/entropy/create-your-first-entropy-app.mdx:162]: I don't think this is necessary.

**Pull request merged**